internal/report/report.go

```diff
@@ -1246,20 +1254,23 @@ func computeTotal(prof *profile.Profile, value, meanDiv func(v []int64) int64) i
 	if diffTotal > 0 {
 		total = diffTotal
 		div = diffDiv
+		// negative samples are expected in diff base profiles.
+		negSamples = false
 	}
 	if div != 0 {
-		return total / div
+		return total / div, negSamples
 	}
-	return total
+	return total, negSamples
 }
 
 // Report contains the data and associated routines to extract a
 // report from a profile.
 type Report struct {
-	prof        *profile.Profile
-	total       int64
-	options     *Options
-	formatValue func(int64) string
+	prof            *profile.Profile
+	total           int64
+	options         *Options
+	formatValue     func(int64) string
+	unexpNegSamples bool
```

Comment:
Maybe move it up to be after "total" since these are related.

Reply:
Done.